### PR TITLE
Update requirements.txt

### PR DESCRIPTION
**Pull request by seanth**
While trying to install on Apple silcon I found that I needed to get the Cython modules installed.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
+Cython
```
Comment (Jeronymous) [inline requirements.txt:1]: Do you understand what is using Cython?
Was this a requirement for whisper already?

File: whisper_timestamped/transcribe.py [hunk 2]
```diff
@@ -630,6 +639,13 @@ def cli():
     parser.add_argument("--model_dir", default=None, help="The path to save model files; uses ~/.cache/whisper by default", type=str)
     parser.add_argument("--device", default="cuda:0" if torch.cuda.is_available() else "cpu", help="device to use for PyTorch inference")
     parser.add_argument("--output_dir", "-o", default=None, help="directory to save the outputs", type=str)
+
+    parser.add_argument("--csv", default=True, help="directory to save the outputs", type=str2bool)
+    parser.add_argument("--json", default=False, help="directory to save the outputs", type=str2bool)
+    parser.add_argument("--srt", default=True, help="directory to save the outputs", type=str2bool)
+    parser.add_argument("--txt", default=True, help="directory to save the outputs", type=str2bool)
+    parser.add_argument("--vtt", default=True, help="directory to save the outputs", type=str2bool)
+
```
Comment (Jeronymous) [inline whisper_timestamped/transcribe.py:648]: Nice addition :+1: 
I like the fact that it's doing (almost) the same as `whisper` by default. And that we can offer more flexibility.

Just, shouldn't the help messages be "whether to save the outputs in XXX format"?
(everything being saved in the same folder specified by --output_dir)

About the json not being an output by default, a note should be added in the README (which shows this output, that is the more complete and well structured)

**Review by Jeronymous — COMMENTED** [review]
Thank you Sean for this PR!

File: requirements.txt (hunk 1 above)
Comment (seanth) [reply requirements.txt:1]: I'm pretty sure it was dtw-python. Whisper on it's own didn't require Cython. It was only when I was trying to use whisper-timestamped

Comment (Jeronymous) [reply requirements.txt:1]: OK thanks a lot.
I merged your PR in master (partially by mistake...), and made the modifications I was suggesting on this PR

**Pull request merged**